heat-stack/app/components/ui/heat/CaseSummaryComponents/AnalysisHeader.tsx

```diff
@@ -58,40 +61,45 @@ export function AnalysisHeader({ usage_data }: { usage_data: UsageDataSchema}) {
 
 
 	return (
-		<div className="section-title">
-			<div className="item-group-title">Heat Load Analysis</div>
-			<div className="flex flex-row">
+		<div className="section-title mt-12">
+			<div className="flex flex-row gap-0.5 mb-4">
+				<div className="item-group-title font-semibold text-2xl">Heat Load Analysis</div>
+				{/* TODO: add help text here */}
+				<img src={HelpCircle} alt='help text'/>
```

Comment:
I suggest commenting out the help circle until we add the help text. Or add the help text before merging this. I feel that's pretty important.

```suggestion
				{/* TODO: add help text here */}
				{/* <img src={HelpCircle} alt='help text'/> */}
```